bff/src/Altinn.Authentication.UI/Altinn.Authentication.UI.Core/SystemUsers/ISystemUserService.cs

```diff
@@ -14,7 +14,7 @@ public interface ISystemUserService
     /// <summary>
     /// Creates a system user
     /// </summary>
-    Task<SystemUser?> CreateSystemUser(int partyId, SystemUserDescriptor newSystemUserDescriptor, CancellationToken cancellation = default);
+    Task<(SystemUser?, string?)> CreateSystemUser(int partyId, SystemUserDescriptor newSystemUserDescriptor, CancellationToken cancellation = default);
```

Comment:
Name tuple arguments - especially when using plain types like `string`.

Reply:
ok, changed to Result type